ui/packages/catalog-ui-search/src/main/webapp/component/input/enum/input-enum.view.js

```diff
@@ -17,24 +17,9 @@ const template = require('./input-enum.hbs')
 const InputView = require('../input.view')
 const DropdownView = require('../../dropdown/dropdown.view.js')
 const moment = require('moment')
-const user = require('../../singletons/user-instance.js')
+const InputUtil = require('../inputUtil.js')
```

Comment:
not sure if there is a better way to do this import so that I can import and call `getEnumValue` directly. my attempts have failed so far.

Reply:
that's only syntactic sugar though right? it still works "as is" though?

Reply:
correct, it works as-is.

Reply:
you could probably do `const getEnumValue = require('../inputUtil.js').getEnumValue` if you want